### ISSUE TITLE
Git push failing after upgrade (ArchLinux)

### ISSUE THREAD

**Issue by qom** [keybase/client#23345]
Firstly, thank you for developing this encrypted git service. I've used it lightly for a couple months now, and it's been working nicely so far.

I'm running ArchLinux. Today I upgraded to a newer version of keybase, made a few commits to my local repo, and upon pushing ran into an error. I tried before and after running the recommended garbage collection and continue to get the same error (removed repo name): 

```
]$ git push
Initializing Keybase... done.
Syncing with Keybase... done.
Preparing and encrypting: (100.00%) 19/19 objects... done.
Indexing hashes: (94.74%) 18/19 objects... done.
Indexing CRCs: (94.74%) 18/19 objects... done.
Indexing offsets: (94.74%) 18/19 objects... done.
To keybase://private/<..snip..>
 ! [remote rejected] master -> master (some refs were not updated)
error: failed to push some refs to 'keybase://private/<..snip..>'
```

I'm not sure if the error is related to updating to a newer version of keybase, but here is the pacman install log with the old and new keybase versions:

```
[2020-03-28T13:01:59-0400] [ALPM] upgraded keybase-bin (5.1.1_20191211223501+15bbb94c23-1 -> 5.3.1_20200320154633+3e235215b3-1)
```
I haven't looked at the log files in detail, but I see many instances of lines that begin like this in `$HOME/.cache/keybase/keybase.git.log`:

```
2020-03-28T21:56:38.001457-04:00 ▶ [DEBU git block_retrieval_queue.go:622] 19e8 Couldn't get block BlockPointer
```

my log id: bdc0ef6182574fa46bdf151c

**Comment by strib** [comment]
Hi!  I don't see anything obviously wrong in the logs (those messages you cited is normal at the end of a run).  Are you certain you don't need to force push?

**Comment by qom** [comment]
Oh boy. Egg on my face here. Thanks very much for the quick response!

I hadn't touched this repo in a bit and thought it was up-to-date with the remote. It turns out it had diverged and I needed to pull and merge some remote changes first (what can I say? It was late). I'm managing this repo via a git UI that's under development so that confused things a bit as well.

I resolved merge conflicts and was able to push. Sorry to make you look through logs for a non-issue.
